railties/lib/rails/application/configuration.rb

```diff
@@ -129,6 +129,7 @@ def load_defaults(target_version)
 
           if respond_to?(:action_dispatch)
             action_dispatch.use_cookies_with_metadata = true
+            action_dispatch.return_only_media_type_on_content_type = false
```

Comment:
If load_defaults "5.2" with upgraded Rails 6.0 app, should this option should be false?

Reply:
I see, nevermind. New default is false, but for older load_defaults apps, setting true by the line https://github.com/rails/rails/pull/36490/files#diff-182aa02288995070b335f57216738b92R26.